ignite-cli/src/index.es

```diff
@@ -0,0 +1,37 @@
+#! /usr/bin/env node
+'use strict'
+import Program from 'commander'
+import colors from 'colors/safe'
+import pjson from './package.json'
+import Shell from 'shelljs'
```

Comment:
The inconsistency here makes me 

![crazy](https://media4.giphy.com/media/oGH4GlC8wAHTy/giphy.gif)